pkg/meta/sql.go

```diff
@@ -1585,6 +1585,7 @@ func (m *dbMeta) doRmdir(ctx Context, parent Ino, name string, pinode *Ino, skip
 		}
 		if _, err := s.Delete(&dirStats{Inode: e.Inode}); err != nil {
 			logger.Warnf("remove dir usage of ino(%d): %s", e.Inode, err)
```

Comment:
We don't need this warning anymore. Please remove it or warp it with the origin error.